dockerfiles/Dockerfile_full

```diff
@@ -0,0 +1,22 @@
+FROM heroku/heroku:18
+
+ENV DEBIAN_FRONTEND=noninteractive
+ARG SALESFORCE_CLI_VERSION=latest
+RUN apt-get update
+RUN echo 'f6d5a254de2734ad9e7b9b02777d47a9a7454059e2d5690cf25c5ea57280f060  ./nodejs.deb' > node-file-lock.sha \
+  && curl -s -o nodejs.deb https://deb.nodesource.com/node_10.x/pool/main/n/nodejs/nodejs_10.16.2-1nodesource1_amd64.deb \
+  && shasum --check node-file-lock.sha
+RUN apt-get install --assume-yes \
+  ./nodejs.deb \ 
+  openjdk-11-jdk-headless=11.0.4+11-1ubuntu2~18.04.3 \
+  jq \
```

Comment:
Added jq 
https://stedolan.github.io/jq/

Reply:
I wonder if Instead of renaming the Dockerfile if we should create a folder called full and move this Dockerfile over there. Same thing with slim. I see some github repos following this pattern. Might be worth looking into see if there are any best practices here that we could follow when we have multiple docker files. https://github.com/DSpace-Labs/DSpace-Docker-Images/tree/master/dockerfiles

Reply:
@praksb I'm open to different patterns, whatever makes the most sense organizationally to us. I have not found any literature sitting any performance or pitfalls when naming the Dockerfiles, just to be aware of the [build context](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). 
In my mind, because we will have no other associated files with the different images, just the Dockerfiles. The organizational goal would be mainly to make sure people associate and edit the correct Docekrfile with regards to `full` and `slim`.
Another naming convention we could is is `<purpose>.Dockerfile` so `full.Dockerfile`.
https://stackoverflow.com/questions/26077543/how-to-name-dockerfiles

Reply:
Based on our conversation earlier, I am going to leave the Dockerfiles as is in this PR and we can easily change the organization of the repo later on if we want to.

Reply:
sounds good.